src/forwarder.ts

```diff
@@ -6,6 +6,17 @@ const logGroupKey = "AWS::Logs::LogGroup";
 const logGroupSubscriptionKey = "AWS::Logs::SubscriptionFilter";
 const maxAllowableLogGroupSubscriptions: number = 2;
 
+/**
+ * Error to be thrown when function ARN cannot be found.
+ */
```

Comment:
Nit: I'm not sure you need this comment here -- it's pretty obvious what the error does even without the comment. And that's a good thing! 🙂